pennylane_lightning/core/src/utils/Util.hpp

```diff
@@ -24,7 +24,7 @@
 #include <numbers>
 #include <numeric> // transform_reduce
 #include <set>
-#include <type_traits> // is_same_v
+#include <type_traits> // integral, floating_point, is_same_v
```

Comment:
I think `std::integral` and `std::floating_point` are in `concepts` 
```suggestion
#include <type_traits> // is_same_v
```
reference: https://en.cppreference.com/w/cpp/header/type_traits

Reply:
Ah, I was looking at `https://en.cppreference.com/w/cpp/types/is_integral` which is type_traits. Ok, I'll add the header.